components/organisms/DashboardHeader.vue

```diff
@@ -59,6 +69,12 @@ import { KYBStatusEnum } from '~/plugins/weavr-multi/api/models/identities/corpo
 import { KYCStatusEnum } from '~/plugins/weavr-multi/api/models/identities/consumers/enums/KYCStatusEnum'
 import { weavrCurrency } from '~/utils/helper'
 
+const route = useRoute()
+
+const isRouteActive = (basePath: string) => {
+    return route.path.startsWith(basePath)
+}
```

Comment:
Should this be in a helper file?

Reply:
Only used in one file, will leave it for now